src/renderer/components/Gallery.tsx

```diff
@@ -140,43 +254,149 @@ export default class Gallery extends Component<
                 </li>
               )
             })}
+            {showDateHeader && (
+              <div className='big-date' ref={this.dateHeader}></div>
+            )}
+            {this.state.id === 'files' && (
+              <>
+                <div style={{ flexGrow: 1 }}></div>
+                <div className='searchbar'>
+                  <input
+                    type='search'
+                    placeholder='search files'
+                    onChange={this.onChangeInput.bind(this)}
+                  />
+                </div>
+              </>
+            )}
           </ul>
-          <div className='bp4-tab-panel' role='tabpanel'>
-            <div className='gallery'>
-              {errors.length > 0 && (
-                <div className='loading-errors'>
-                  The following messages failed to load, please report these
-                  errors to the developers:
-                  <ul>
-                    {errors.map(error => (
-                      <li key={error.msgId}>
-                        {error.msgId} {'->'} {error.error}
-                      </li>
-                    ))}
-                  </ul>
+          <div role='tabpanel'>
+            <div
+              className={`gallery gallery-image-object-fit_${
+                GalleryImageKeepAspectRatio ? 'contain' : 'cover'
+              }`}
+              key={this.state.msgTypes.join('.') + String(this.props.chatId)}
+            >
+              {mediaMessageIds.length < 1 && !loading && (
+                <div className='empty-screen'>
+                  {/* IDEA: when we have someone doing illustrations this would be a great place to add some */}
```

Comment:
How about we open an issue for this and add "help wanted/needed" label?

Reply:
would get closed immediately because we don't have an designer/illustrator, I think we can keep this comment for now.